woocommerce-services.php

```diff
@@ -651,12 +652,17 @@ public function woocommerce_payment_gateways( $payment_gateways ) {
 		 * Registers the React UI bundle
 		 */
 		public function admin_enqueue_scripts() {
-			wp_register_style( 'noticons', plugins_url( 'assets/stylesheets/noticons.css', __FILE__ ), array(), '20150727' );
-			wp_register_style( 'dashicons', plugins_url( 'assets/stylesheets/dashicons.css', __FILE__ ), array(), '20150727' );
+			// Note: This will break outside of wp-admin, if/when we put user-facing JS/CSS we'll have to figure out another way to version them
```

Comment:
> ... allows us to update versions only for scripts and styles that change between releases.

That's not a big deal. Realistically speaking, we are going to update the main JS bundle on every release.

> It also doesn't require plugin file header parsing on every admin request.

That's not a performance problem, something in the wp-admin code already parses *all* the plugin headers on every request.

> We could just hardcode a date-based version string for each registered script and style. For example, a version of 20170218 or 1487454198.

The problem with that is that we will forget. Using the same strategy as WooCommerce (`define 'WC_VERSION', '1.2.3')`) is a little less error-prone because it's just one value. Even then, I would prefer to fully automate this while possible (that is, until we start servin JS/CSS in non-admin pages), and parsing the plugin info is the most straight-forward way to do that.